embed/fs.go

```diff
@@ -0,0 +1,138 @@
+// Package embed contains an embedded copy of the flux standard library.
+// It exposes a filesystem implementation that can be used to read from
+// an already compiled copy of the standard library that can then be
+// loaded at runtime as the semantic graph.
+//
+// The compiled versions of the semantic graph are semantic graphs
+// marshaled as flatbuffers and then gzipped. In order to read them,
+// the filesystem reads the gzip file and then deserializes the
+// flatbuffer as a semantic graph.
```

Comment:
Is using flatbuffers here a new code path? I don't think it is but wanted to confirm. What I am concerned about is we have a latent bug in how we encode semantic graph as flatbuffers that this change might expose.

Reply:
It's not a new code path. I believe this invokes a rust function to marshal the semantic graph.